### PR TITLE
Feature/merge stickers models

### PR DESCRIPTION
**Pull request by Kiruha01**

### CODE REVIEW

File: pybotx/bot/bot.py [hunk 1]
```diff
@@ -1231,7 +1231,7 @@ async def add_sticker(
 
         botx_api_sticker = await method.execute(payload)
 
-        return botx_api_sticker.to_domain()
+        return botx_api_sticker.to_domain(sticker_pack_id)
```
Comment (Nimond) [inline pybotx/bot/bot.py:1234]: `pack_id=`

File: pybotx/models/message/incoming_message.py [hunk 2]
```diff
@@ -7,11 +7,10 @@
 
 from pybotx.logger import logger
 from pybotx.models.async_files import APIAsyncFile, File, convert_async_file_to_domain
-from pybotx.models.attachments import (
+from pybotx.models.attachments import (  # AttachmentSticker,
```
Comment (lxmnk) [inline pybotx/models/message/incoming_message.py:10]: Forgot to remove.

File: tests/test_attachments.py [hunk 3]
```diff
@@ -140,11 +140,11 @@ async def default_handler(message: IncomingMessage, bot: Bot) -> None:
                 "pack": "4e4cfd0b-b981-54e9-84f4-3ddc12600334",
             },
         },
-        AttachmentSticker(
-            type=AttachmentTypes.STICKER,
+        Sticker(
             id=UUID("0dfd7318-2ccc-5384-b0e4-6fa5478606a5"),
             image_link="/uploads/sticker_pack/c81e87be5c7949b3b4196769e1032d5f.png",
             pack_id=UUID("4e4cfd0b-b981-54e9-84f4-3ddc12600334"),
+            emoji="/hello",
```
Comment (lxmnk) [inline tests/test_attachments.py:147]: Pls add `body` as first argument into `api_incoming_message_factory` and pass an emoji there.

File: pybotx/bot/bot.py (hunk 1 above)
Comment (Kiruha01) [reply pybotx/bot/bot.py]: Done

File: pybotx/models/message/incoming_message.py (hunk 2 above)
Comment (Kiruha01) [reply pybotx/models/message/incoming_message.py]: Done

File: tests/test_attachments.py (hunk 3 above)
Comment (Kiruha01) [reply tests/test_attachments.py]: Done

**Review by Nimond — APPROVED** [review]

**Review by lxmnk — APPROVED** [review]

**Pull request merged**